dev/io.openliberty.pages.4.0.internal.factories/bnd.bnd

```diff
@@ -0,0 +1,98 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 17
+javac.target: 17
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=17))"
+
+Bundle-Name: Open Liberty Pages 4.0 Factories
+Bundle-SymbolicName: io.openliberty.pages.4.0.internal.factories
+Bundle-Description: WAS Pages 4.0, version ${bVersion}
+
+WLP-ServerName: IBM WebSphere Application Server
+WLP-ServerVersion: ${liberty.beta.version}
+
+-dsannotations: com.ibm.ws.jsp23.webcontainerext.JspVersionFactoryImpl, \
+com.ibm.ws.jsp23.webcontainerext.ws.PrepareJspHelper23Factory, \
+com.ibm.ws.jsp23.translator.visitor.validator.ElValidatorExtFactoryImpl, \
+com.ibm.ws.jsp23.translator.visitor.generator.GeneratorUtilsExtFactoryImpl
+
+# Note these factories were copied from the  io.openliberty.pages.3.1.internal.factories bundle
+
+Export-Package: \
+	com.ibm.ws.jsp.speclevel, \
+	com.ibm.ws.jsp23.webcontainerext;version="4.0", \
+	com.ibm.ws.jsp23.webcontainerext.ws;version="4.0", \
+	com.ibm.ws.jsp23.translator.visitor.validator;version="4.0", \
+	com.ibm.ws.jsp23.translator.visitor.generator;version="4.0", \
+	org.apache.jasper;version="4.0", \
+	org.apache.jasper.el;thread-context=true;version="4.0", \
+	org.apache.jasper.runtime;thread-context=true;version="4.0"
+
+# TODO Update el, servlet, and pages API versions once released
```

Comment:
Follow up: Update API version imports within this bundle once APIs are released.

Reply:
It's here https://github.com/OpenLiberty/open-liberty/issues/26734

Reply:
Thanks, I linked to the Pages 4.1 epic.